tests/v2/validation/pipeline/qase/defaults.go

```diff
@@ -0,0 +1,8 @@
+package qase
+
+const (
+	RancherManagerProjectID = "RM"
+	QaseTokenEnvVar         = "QASE_AUTOMATION_TOKEN"
+	TestRunEnvVar           = "QASE_TEST_RUN_ID"
+	TestRunNAMEEnvVar       = "TEST_RUN_NAME"
```

Comment:
Is this going to be a static name in Jenkins or a dynamically generated environment variable, 
Thinking on after creating a large number of test runs might be a good idea to include other info in the title to be easy to identify visually when navigating on Qase? 

I might be missing where this is coming from.. JJB ?

Reply:
This is coming from JJB/Jenkins. These are the env vars needed. Does that make sense?